src/ProjNet/ProjNET.csproj

```diff
@@ -30,7 +30,6 @@ Proj.NET performs point-to-point coordinate conversions between geodetic coordin
   </ItemGroup>
 
 	<ItemGroup Condition=" '$(EnableApiCompat)' == 'true' ">
-		<PackageReference Include="Microsoft.DotNet.ApiCompat" Version="6.0.0-beta.21159.11" PrivateAssets="All" />
```

Comment:
Why did you remove this? Have you tried to update the package?
The functionality it provides should definetly stay in place.

Reply:
I can just assume that he experienced the same problem described [here](https://github.com/NetTopologySuite/NetTopologySuite/pull/580#discussion_r781839363). anyway, same reply is still valid.

Reply:
Yup, @DGuidi is right. In order to get the project to compile, I had to remove the reference. I didn't mean to include it in this commit though, that's my mistake.
I can't find that nuget package anywhere on nuget.org, including pre-releases. Where do I restore that package from?

Reply:
Good catch, it will need to already have been in your package cache in order for this to work.

We should have added a NuGet.config file like what NTS has: https://github.com/NetTopologySuite/NetTopologySuite/blob/54ff4b3a21ded301159d12277cf8515b463f3394/NuGet.config#L1-L6